client.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys
+import time
+
+from tarantool.connection import Connection
+from tarantool.error import NetworkError
+from tarantool.utils import ENCODING_DEFAULT
+
+class RoundRobinStrategy(object):
+    def __init__(self, addrs):
+        self.addrs = addrs
+        self.pos = 0
+
+    def getnext(self):
+        tmp = self.pos
+        self.pos = (self.pos + 1) % len(self.addrs)
+        return self.addrs[tmp]
+
+class MeshConnection(Connection):
+    def __init__(self, addrs, user, password, connect_now, encoding, Strategy):
+        self.nattempts = 2 * len(addrs) + 1
+        self.Strategy = Strategy(addrs)
+        addr = self.Strategy.getnext()
+        host = addrs[0]['host']
+        port = addrs[0]['port']
+        super(MeshConnection, self).__init__(host=host,
+                                             port=port,
+                                             user=user,
+                                             password=password,
+                                             connect_now=connect_now,
+                                             encoding=encoding)
+
+    def _opt_reconnect(self):
+        nattempts = self.nattempts
+        while nattempts > 0:
+            try:
+                super(MeshConnection, self)._opt_reconnect()
+                break
+            except NetworkError:
+                nattempts -= 1
+                addr = self.Strategy.getnext()
+                self.host = addr['host']
+                self.port = addr['port']
+        else:
+            raise NetworkError
+
+def connectmesh(addrs=({'host': 'localhost', 'port': 3301},), user=None,
+                password=None, connect_now=True, encoding=ENCODING_DEFAULT,
+                Strategy=RoundRobinStrategy):
+    '''
+    Create a connection to the Tarantool server.
+
+    :param str host: Server hostname or IP-address
+    :param int port: Server port
+
+    :rtype: :class:`~tarantool.connection.Connection`
+
+    :raise: `NetworkError`
+    '''
+
+    return MeshConnection(addrs=addrs,
+                          user=user,
+                          password=password,
+                          connect_now=connect_now,
+                          encoding=encoding,
+                          Strategy=Strategy)
+
+addrs=(
```

Comment:
I think we should not place some hardcoded actions like these ones (here and below) in the module.

Reply:
Esp. when they can fail because of external conditionals (whether are tarantools running on these ports).